foxy/distribution.yaml

```diff
@@ -3612,6 +3612,24 @@ repositories:
       url: https://github.com/ros2/rosidl_typesupport_gurumdds.git
       version: foxy
     status: developed
+  rosxbee:
+    doc:
+      type: git
+      url: https://github.com/Sudharsan10/ROSXBee.git
+      version: 0.0.1
+    release:
+      packages:
+      - rosxbeepy
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/Sudharsan10/ROSXBee-release.git
+      version: 0.0.1-2
+    source:
+      test_pull_requests: true
+      type: git
+      url: https://github.com/Sudharsan10/ROSXBee.git
+      version: 0.0.1
```

Comment:
same here

Reply:
@ivanpauno  Fixed the version to branch on both places mentioned